lib/generators/rails_bootstrap/controllers_generator.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails/generators'
+
+module RailsBootstrap
+  module Generators
+    class ControllersGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../../..", __FILE__)
+
+      def copy_components
```

Comment:
what does this do?

Reply:
This is where all our javascript is organized in the stimulus. Javascript classes in Stimulus is called Controller. Instance od controller can be attached to any DOM element. You can then manupilate that particular DOM element using the attached controller. I would higley suggest you to read: 

1. https://stimulusjs.org/handbook/origin
2. https://www.smashingmagazine.com/2020/07/introduction-stimulusjs/

Reply:
Also, if you were asking about what this generator do, then it will copy all the controller we define in rails_bootstrap's `app/javascript/controllers` folder to the app's (which uses our engine)`app/javascript/controllers` folder.

For that people have to type: `rails g rails_bootstrap:controllers`